tests/integration/test_grakn.py

```diff
@@ -23,7 +23,7 @@
 from grakn import GraknClient, ValueType, Transaction
 from grakn import GraknError
 
-from test.integration.base import test_base, GraknServer
+from tests.integration.base import test_base, GraknServer
```

Comment:
any particular reason for this @lolski ?

Reply:
Yes, according to @alexjpwalker , we can't use the word `test` because there's a built-in module with the same name that causes the runtime to be confused. Is that correct Alex?

Reply:
:))))))))))))

Reply:
Python is fun isn't it

Reply:
Incidentally the reason Ganesh had to make this change is because I forgot to (was planning to delete these tests in the future once any useful parts are migrated)